test/e2e/egress_services.go

```
@@ -1002,15 +1002,15 @@ spec:
 				}, 1*time.Second, 200*time.Millisecond).ShouldNot(gomega.HaveOccurred(), "failed to reach external container with node's ip")
 			}
 		},
-		ginkgo.Entry("ipv4 pods", v1.IPv4Protocol),
-		ginkgo.Entry("ipv6 pods", v1.IPv6Protocol),
+		ginkgo.Entry("ipv4 pods", corev1.IPv4Protocol),
+		ginkgo.Entry("ipv6 pods", corev1.IPv6Protocol),
 	)
 
 	ginkgo.DescribeTable("[LGW] Should validate ingress reply traffic uses the Network",
-		func(protocol v1.IPFamily, isIPv6 bool) {
+		func(protocol corev1.IPFamily, isIPv6 bool) {
```

golangci-lint (1.64.8)

[warning] unused-parameter: parameter 'isIPv6' seems to be unused, consider removing or renaming it as _

(revive)